### PR TITLE
Move from Travis to GH actions

### PR DESCRIPTION
**Pull request by banaag**
Move presubmit infra from travis to Github actions.

### CODE REVIEW

File: transformer/transformer_test.go [hunk 1]
```diff
@@ -309,8 +309,8 @@ func TestCustom(t *testing.T) {
 }
 
 func TestCustomFail(t *testing.T) {
-	r := rpb.Request{Html: "<html ⚡><lemur>", Config: rpb.Request_CUSTOM, Transformers: []string{"does_not_exist"}}
-	if html, _, err := Process(&r); err == nil {
+	r := &rpb.Request{Html: "<html ⚡><lemur>", Config: rpb.Request_CUSTOM, Transformers: []string{"does_not_exist"}}
```
Comment (twifkak) [inline transformer/transformer_test.go:312]: OOC, what was the error/warning that precipitated this change?

**Review by twifkak — CHANGES_REQUESTED** [review]

File: .github/workflows/prerequisites.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      uses: actions/setup-go@v1
+      with:
+        go-version: '1.14'
+
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+
+    - name: Diagnose the code with go vet
+      run: go vet ./...
+
```
[the remark this answers is not in the document]
Comment (banaag) [reply .github/workflows/prerequisites.yml:41]: Added.

File: transformer/transformer_test.go (hunk 1 above)
Comment (banaag) [reply transformer/transformer_test.go:312]: transformer/transformer_test.go:314:49: call of t.Fatalf copies lock value: github.com/ampproject/amppackager/transforme
r/request.Request contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex

Comment (twifkak) [reply transformer/transformer_test.go:312]: Ah, huh, good to know, thx.

File: .github/workflows/prerequisites.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    steps:
+    - name: Setup Go 1.16
+      uses: actions/setup-go@v1
+      with:
+        go-version: '1.16'
+
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+
+    - name: Diagnose the code with go vet
```
[the remark this answers is not in the document]
Comment (banaag) [reply .github/workflows/prerequisites.yml]: Oops, fixed.

**Pull request merged**